### PR TITLE
mpc85xx: Add testing kernel for mpc85xx

### PR DESCRIPTION
**Pull request by CHKDSK88**
This patch series introduces 6.1 testing kernel for mpc85xx architecture.

### CODE REVIEW

**Review by neheb — APPROVED** [review]

File: target/linux/mpc85xx/image/p1010.mk [hunk 1]
```diff
@@ -21,8 +21,8 @@ define Device/aerohive_br200-wp
   KERNEL_NAME := simpleImage.br200-wp
   KERNEL := kernel-bin | uImage none
   KERNEL_INITRAMFS := kernel-bin | uImage none
-  KERNEL_ENTRY := 0x1000000
-  KERNEL_LOADADDR := 0x1000000
+  KERNEL_ENTRY := 0x1500000
+  KERNEL_LOADADDR := 0x1500000
```
Comment (hauke) [inline target/linux/mpc85xx/image/p1010.mk:25]: Is it possible to change the  addresses only here? On many other targets you have to change the load address in uboot too and they have to match. I am not familiar with the target,

Comment (CHKDSK88) [reply target/linux/mpc85xx/image/p1010.mk:25]: In mxp85xx it works quite differ.

Please take a look TP-Link TL-WDR4900 v1 bootcmd:
`sf probe 0;sf read 2000000 60200 69f000;sf read 3000000 50000 10000;bootm 2000000 - 3000000`
BR200-WP loads kernel directly from NOR. 

Mpc85xx u-boot unpack (or just relocate uncompressed, like in this case) image into `KERNEL_LOADADDR` and boot from `KERNEL_ENTRY`. It loads dedicated mpc85xx loader, which unpack lzma compressed kernel into address 0x0. At the end of process, kernel is starting from address 0x0 without any arguments from u-boot (mpc85xx SimpleImage method).

I run tested it on WDR4900 and BR200-WP:
```
[    0.000000] Linux version 6.1.62 (a@b) (powerpc-openwrt-linux-musl-gcc (OpenWrt GCC 13.2.0 r24439-7e50a9f565dd) 13.2.0, GNU ld (GNU Binutils) 2.41) #0 Tue Nov 21 09:50:53 2023
[    0.000000] Hardware name: TP-Link TL-WDR4900 v1 e500v2 0x80212151 Freescale P1014
```

**Comment by chunkeey** [comment]
Thank you. I have a Aerohive AP330/AP350 and I can confirm that yes, this works.

Though, the Aerohive gave me quite the headache since the 6.1 kernel grew over the 16MiB. And kicking down the bucket and just increasing `KERNEL_SIZE` no longer worked. I think I have the simpleImage conversion for the device "ready". The question is, should I add it before or after switching mpc85xx to 6.1?

**Review by chunkeey — APPROVED** [review]

**Comment by CHKDSK88** [comment]
@chunkeey 
>  I think I have the simpleImage conversion for the device "ready". The question is, should I add it before or after switching mpc85xx to 6.1?

IMO the better time slot will be after this PR and before switch to 6.1.

**Comment by chunkeey** [comment]
Merged. Thanks!

**Pull request closed without merging**